packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts

```diff
@@ -235,19 +234,7 @@ export class AuditLogManager {
         error: {
           message: 'Failed to generate the audit logs CSV',
         },
-        ok: null,
       };
     }
   }
-
-  async maskTokenForAuditLog(token: string, visibleStart = 3, visibleEnd = 3): Promise<string> {
-    if (token.length <= visibleStart + visibleEnd) {
-      return token;
-    }
-    return (
-      token.slice(0, visibleStart) +
-      '*'.repeat(token.length - visibleStart - visibleEnd) +
-      token.slice(-visibleEnd)
-    );
-  }
```

Comment:
There is no need to re-declare this function here if it is already available everywhere by importing from `@hive/service-common`